src/commands/natural-chat.ts

```
@@ -29,17 +29,18 @@ export default function naturalChatCommand(program: Command) {
       const session: ChatSession = {
         messages: [],
         sessionId: generateSessionId(),
-        startTime: new Date()
+        startTime: new Date(),
       };
 
       // チャットループ
-      while (true) {
+      let continueChat = true;
```

ESLint

[error] 'continueChat' is never reassigned. Use 'const' instead.

(prefer-const)